examples/getstarted/src/api/kitchensink/content-types/kitchensink/schema.json

```diff
@@ -134,6 +134,11 @@
     "custom_field": {
       "type": "customField",
       "customField": "plugin::color-picker.color"
+    },
+    "custom_field_with_default_options": {
+      "type": "customField",
+      "regex": "^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$",
```

Comment:
Shouldn't this be in an "options" key like we discussed a while back?

Reply:
I think not because that is a [root level option](https://github.com/strapi/strapi/blob/c8888bf2cfaedcb1c94bad9ffcbe531d4d71a834/packages/core/admin/admin/src/core/apis/CustomFields.js#L22) that comes with the underlying data type:

[I check the options](https://github.com/strapi/strapi/blob/c8888bf2cfaedcb1c94bad9ffcbe531d4d71a834/packages/core/admin/admin/src/core/apis/CustomFields.js#L48) and throw an error if an option is not allowed at the root level.

Reply:
Makes sense I thought it was specific the the field 👍